src/utilities/util_main.c

```diff
@@ -289,6 +297,9 @@ main(int argc, char *argv[])
         goto err;
     }
 
+    if (recover || salvage)
```

Comment:
I'd like a comment here talking about how salvage and recovery override the readonly option. It is not immediately obvious.